### PR TITLE
Add _include to DSTU2 Slot search

### PR DESCRIPTION
**Pull request by FFSol**

### CODE REVIEW

File: lib/resources/example_json/dstu2_examples_slot.rb [hunk 1]
```diff
@@ -286,5 +286,463 @@ module Resources
...
+                    "code": "394581000",
+                    "display": "Community medicine"
+                  }
+                ],
+                "text": "Same Day"
+              }
+            ],
+            "actor": {
+              "reference": "Practitioner/2578010",
+              "display": "Howdeshell, Tami"
```
Comment (cerbs) [inline lib/resources/example_json/dstu2_examples_slot.rb]: Replace all of these displays with a made up person (and in the text.div field

Comment (FFSol) [reply lib/resources/example_json/dstu2_examples_slot.rb]: https://github.com/cerner/fhir.cerner.com/pull/256/commits/42269d94e855b35f116fba9fccc9feff6eada449

Comment (FFSol) [reply lib/resources/example_json/dstu2_examples_slot.rb]: Oops, accidentally updated the Pract ID too. Reverted https://github.com/cerner/fhir.cerner.com/pull/256/commits/7d79cb17af18d809a419eb3dfb0ddc316adcbd5a

File: content/millennium/dstu2/scheduling/slot.md [hunk 2]
```diff
@@ -94,6 +95,12 @@ Notes:
 
 * The search operation will only return free slots which are available to be booked. However, booked slots may still be retrieved when using the `_id` parameter.
 
+* The `_include` parameter may be provided once. Example: `_include=Slot:schedule`
```
Comment (MaxPhilips) [inline content/millennium/dstu2/scheduling/slot.md]: mention that Slot:schedule is the only supported value

File: content/millennium/dstu2/scheduling/slot.md [hunk 3]
```diff
@@ -94,6 +95,12 @@ Notes:
...
+
+* The `_include` parameter may be provided with the `_id` parameter. Example: `_id=21265426-633867-3121665-0&_include=Slot:schedule`
+
+* Additional OAuth2 scopes are required when the `_include` parameter is provided. To include with `Slot:schedule`, the OAuth2 token must include the scope for Schedule.read.
```
Comment (MaxPhilips) [inline content/millennium/dstu2/scheduling/slot.md]: reword to match above given only the one value is allowed. maybe:

When `_include` is provided in a request to the closed endpoint, the OAuth2 token must include either the `user/Schedule.read` or the `patient/Schedule.read` scope in addition to a Slot scope.

File: content/millennium/dstu2/scheduling/slot.md (hunk 2 above)
Comment (FFSol) [reply content/millennium/dstu2/scheduling/slot.md]: https://github.com/cerner/fhir.cerner.com/pull/256/commits/eb14ba57695d47945c5d478cd6afdf31bf97b788#diff-5af5ea7f1f259fa879893204fdd065b5R98

File: content/millennium/dstu2/scheduling/slot.md (hunk 3 above)
Comment (FFSol) [reply content/millennium/dstu2/scheduling/slot.md]: https://github.com/cerner/fhir.cerner.com/pull/256/commits/eb14ba57695d47945c5d478cd6afdf31bf97b788#diff-5af5ea7f1f259fa879893204fdd065b5R102

**Comment by FFSol** [comment]
Is there anything left to get this merged?

**Comment by FFSol** [comment]
@cerbs @beichhor @jfrhong ^^

**Pull request merged**